Binary files mx4j-2.1.1/build/.build.xml.swp and mx4j-2.1.1-new/build/.build.xml.swp differ
diff -Nurd mx4j-2.1.1/build/build.xml mx4j-2.1.1-new/build/build.xml
--- mx4j-2.1.1/build/build.xml	2005-01-16 15:58:23.000000000 -0500
+++ mx4j-2.1.1-new/build/build.xml	2010-01-29 13:03:53.000000000 -0500
@@ -170,12 +170,6 @@
    </target>
 
    <target name="flags" description="Sets the flags for conditional compilation">
-      <!-- JDK 1.4 flag -->
-      <condition property="jdk14.present">
-         <available classname="java.util.logging.Logger"/>
-      </condition>
-      <antcall target="jdk14-present"/>
-
       <!-- JAAS flag -->
       <condition property="jaas.present">
          <available classname="javax.security.auth.Subject" classpathref="lib.classpath"/>
@@ -196,14 +190,6 @@
          <available classname="org.python.util.PythonInterpreter" classpathref="lib.classpath"/>
       </condition>
       <antcall target="jython-present"/>
-
-      <!-- Jetty flag -->
-      <condition property="jetty.present">
-         <and>
-            <available classname="org.mortbay.jetty.Server" classpathref="lib.classpath"/>
-         </and>
-      </condition>
-      <antcall target="jetty-present"/>
    </target>
 
    <target name="releasing">
@@ -458,6 +444,7 @@
       <unzip src="${docbook.xsl.zip.file}" dest="${docs.src.dir}/docbookx"/>
       <mkdir dir="${dist.docs.dir}"/>
 
+      <property name="javax.xml.transform.TransformerFactory" value="com.icl.saxon.TransformerFactoryImpl" />
       <style processor="trax"
          basedir="${docs.src.dir}"
          destdir="${dist.docs.dir}"
@@ -465,7 +452,8 @@
          style="${docs.src.dir}/xsl/mx4j-chunk.xsl"
          includes="${root.document.name}"
          classpathref="lib.classpath"
-         />
+         >
+      </style>
 
       <delete dir="${docs.src.dir}/docbook" quiet="true"/>
       <delete dir="${docs.src.dir}/docbookx" quiet="true"/>
